doc/website/release-notes/iceoryx-v2-0-0.md

```diff
@@ -6,13 +6,19 @@
 
 **Features:**
 
+- block publisher when subscriber queue full [\#615](https://github.com/eclipse-iceoryx/iceoryx/issues/615)
+- extend chunk header for gateways and recompute [\#711](https://github.com/eclipse-iceoryx/iceoryx/pull/711)
+- extend waitset and listener for member use case and waitset callback for user type [\#707](https://github.com/eclipse-iceoryx/iceoryx/issue/707)
```

Comment:
I'm pretty sure these were already in iceoryx 1.0 but we added these features on the release branch first and then backported them